### PR TITLE
fix: correct dependencies and env management

### LINKED ISSUE #311: Update disease-normalizer version
Currently, main points to 0.2.15, but we should update to 0.2.16. We should try to not pin the exact version, but only allow the 0.2.x versions (should do this for ga4gh.vrsatile.pydantic too).

### PR DESCRIPTION
**Pull request by jsstevenson**
close #311. Also handle some other dependency compatibility issues and the main errors in #314 (but leaving that issue open to add documentation/maybe tweak CLI options). Also add the GtoP data extraction update that was merged to the dev branch earlier.

### CODE REVIEW

**Comment by jsstevenson** [comment]
....uh, no idea what is going on with the CI checks

File: Pipfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ boto3 = "*"
 
 [dev-packages]
 thera-py = {editable = true, path = "."}
-disease-normalizer = "==0.2.15"
+disease-normalizer = "~=0.2.16"
```
Comment (korikuzma) [inline Pipfile:16]: Should also do the same for ga4gh.vrsatile.pydantic!

File: setup.cfg [hunk 2]
```diff
@@ -30,7 +30,7 @@ test =
     pytest-cov
 
 dev =
-    disease-normalizer == 0.2.15
+    disease-normalizer ~=0.2.16
```
Comment (korikuzma) [inline setup.cfg:33]: Same comment as above, do the same for ga4gh.vrsatile.pydantic

File: therapy/cli.py [hunk 3]
```diff
@@ -79,6 +79,8 @@ def update_normalizer_db(normalizer: str, aws_instance: str, db_url: str,
             environ[AWS_ENV_VAR_NAME] = aws_env_name
             confirm_aws_db_use(aws_env_name.upper())
             environ[SKIP_AWS_DB_ENV_NAME] = "true"  # this is already checked above
+            endpoint_url = ""
+            db = Database()
```
Comment (korikuzma) [inline therapy/cli.py:83]: You could either delete this + move line 91 outside the else condition or you can change line 83 to `db = Database(db_url=endpoint_url)`

**Pull request closed without merging**